File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao18.py
valor_divida = float(input("Digite o valor da dívida: "))
valor_crescimento = 0
parcela = 0
print("Valor da Dívida      Valor dos Juros         Quantidade de Parcelas        Valor da Parcela")

for i in range(0,13 , 3):
    if i == 0:
        divida = valor_divida
        parcela = i + 1
        juros = 0
        valor_parcela = valor_divida
    else:
        juros = valor_divida * (10 + 5 * valor_crescimento) / 100
        parcela = i
        divida = valor_divida + juros
        valor_crescimento += 1
        valor_parcela = divida / parcela

    
    print(f"    R$ {divida:.2f}                R$ {juros:>6.2f}                  {parcela:<2}                   R${valor_parcela:.2f}")

    
    

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Java_Script/questao18.js
let valor_divida = parseFloat(prompt("Digite o valor da dívida: "))
let valor_crescimento = 0
let parcela = 0
let divida = valor_divida
console.log("Valor da Dívida      Valor dos Juros         Quantidade de Parcelas        Valor da Parcela")

for (i = 0; i < 13; i += 3){
    if (i == 0) {
        parcela = i + 1
        var juros = 0
        var valor_parcela = valor_divida
    }else{
        juros = divida * (10 + 5 * valor_crescimento) / 100
        parcela = i
        divida += juros
        valor_crescimento += 1
        valor_parcela = divida / parcela
    }
    console.log(`R$ ${divida.toFixed(2).padEnd(20)}\t\$ ${juros.toFixed(2).padEnd(20)}\t\t${parcela.toString().padEnd(20)}R$ ${valor_parcela.toFixed(2).padEnd(20)}`)
}